auacm/app/static/html/viewProblem.html

```diff
@@ -3,42 +3,54 @@
     <div class="row">
       <div class="panel panel-default">
         <div class="panel-heading">
-          <h1 class="text-center">{{ current_prob.name }}</h1>
-        </div>
-        <div class="panel-body">
-          <!-- Submit button -->
-          <a href="/#/judge?problem={{ problem.pid }}">
-            <button class="btn btn-success"
-              style="margin-top:2%; width:10%; margin-left:45%;">Submit</button>
-          </a>
-          <br>
+          <div class="row">
+            <h1 class="panel-title col-md-10" style="font-size:32px;">
+              <a href="/problems/{{ current_prob.shortname }}/info.pdf"
+                  target="_blank">
+                {{ current_prob.name }}
+              </a>
+            </h1>
 
-          <!-- Edit button (admins only) -->
-          <a ng-show="isAdmin === 1"
-            href="/#/problems/{{ pid }}/edit">
-            <button class="btn btn-info"
-              style="margin-top:2%; width:10%; margin-left:45%;">Edit</button>
-          </a>
-          <br>
+            <!-- Show the button for submission if the user is not an admin -->
+            <div class="col-md-2">
+              <button ng-if="!isAdmin" class="btn btn-success"
+                style="width:100%;">Submit</button>
+            </div>
 
+            <!-- Show dropdown for submission if the user is an admin -->
+            <div ng-if="isAdmin" class="col-md-2">
+              <button class="btn btn-default dropdown-toggle" style="width:100%;"
+                  type="button" id="dropdownMenu1" data-toggle="dropdown"
+                  aria-haspopup="true">
+                Options
+                <span class="caret"></span>
+              </button>
+              <ul ng-if="isAdmin" class="dropdown-menu" aria-labelledby="dropdownMenu1">
+              <li><a href="/#/judge?problem={{ problem.pid }}">Submit</a></li>
+                <li><a href="/#/problems/{{ pid }}/edit">Edit</a></li>
+              </ul>
```

Comment:
Tbh, I'm not crazy about this. I don't like having to select "Submit" from a dropdown, even if it's just admins.